kmip/core/factories/secrets.py

```diff
@@ -68,7 +69,7 @@ def create(self, secret_type, value=None):
             SymmetricKey(...)
         """
         if secret_type is ObjectType.CERTIFICATE:
-            return self._create_certificate(value)
+            return self._create_certificate()
```

Comment:
Yep, the factory here would just be an extra (unnecessary) step. You can literally do:

    Certificate(certificate_type=CertificateTypeEnum.X_509, certificate_value=b'\x00\x01\x02...')
to create the certificate. It builds everything internal automatically.